### DISCUSSION TITLE
2022-01: Xuanwo's Note 上线啦

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#316] [Announcements]
# 2022-01: Xuanwo's Note 上线啦

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-01/

**Comment by zjuyk** [comment]
好想法，值得学习，我也想整一个

**Reply by Xuanwo** [reply]
整！资瓷！

**Comment by ethinx** [comment]
我是在单独的仓库里面存静态文件，那应该能先用这个action导出，然后commit到web仓库，然后vercel自动部署。周末试试看

**Reply by xiang578** [reply]
上面这个 action 会将生成的文件保存到 gh-page 分支，vercel 直接部署这个分支就可以了。

**Reply by Xuanwo** [reply]
> 上面这个 action 会将生成的文件保存到 gh-page 分支，vercel 直接部署这个分支就可以了。

嗯嗯，我就是这么用的～

**Reply by ethinx** [reply]
因为包括 journal 还有一些工作相关的内容都在一个库里面，所以页面不是默认开放的，就通过私有仓库+vercel 方式部署了。

**Comment by Zheaoli** [comment]
双链好东西啊

**Comment by pengx17** [comment]
学习一下workflow👍

**Comment by GarinZ** [comment]
妙啊！

**Comment by STRRL** [comment]
唔, 所以 xuanwo 也是通过 git 仓库来多端同步 logseq 的吗?

**Reply by Xuanwo** [reply]
没，我只在电脑上写笔记，只 push 从来不 pull :rofl:

**Comment by azusachino** [comment]
老哥，我想咨询一下，logseq中page的markdown格式，是只支持list吗？没有header以及其他模块的话，感觉很难用欸

**Reply by Xuanwo** [reply]
logseq 主打的就是无限嵌套的层级概念，所以界面整体看上去就是一个 list。不过 logseq 是支持 header，code block，加粗，斜体这样的格式的（渲染在行内）～

**Reply by azusachino** [reply]
老哥这大半年来，觉得logseq究竟咋样啊？我以前也就是简单地以md形式记录了不少东西，但后面基本上没有去回顾这些内容；
最近看了不少PKM相关的内容，觉得是时候整理一下“遗产”了，但也还不知道究竟应该用哪个工具。
老哥有践行 Zettelkasten 吗，感觉大致都是积累“原子”的笔记，通过添加tag和reference产生graph；但最终我们回顾的时候会一个点一个点去追溯吗？（虽然这样的图看着挺酷的

**Reply by Xuanwo** [reply]
> 老哥这大半年来，觉得logseq究竟咋样啊？

挺好的，目前唯一感觉不太够了就是 Publish 比较弱

> 老哥有践行 Zettelkasten 吗，感觉大致都是积累“原子”的笔记，通过添加tag和reference产生graph；但最终我们回顾的时候会一个点一个点去追溯吗？

我没有刻意的去遵循某种模式，just writing. 需要找什么东西的时候都是先通过搜索找到一个相近的东西，然后不断的 follow link

**Reply by azusachino** [reply]
其实还是有点担心现在“流行”的这些工具，究竟能运营到什么时候的。
真正开始做PKM的话，肯定会想一直持续下去的；如果时不时需要推倒重来 (切换工具) 的话，还是比较头疼的。
总之谢谢老哥回复，我先试试logseq适不适合，至少本地的这种比notion那类要保守一点。

**Comment by l2dy** [comment]
可以分享一下 Iterations 的自动化是怎么实现的吗？

**Reply by Xuanwo** [reply]
https://github.com/Xuanwo/auto-add-to-project
